health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/appConfigurationRedesign/NavigationLogicWrapper.js

```
@@ -0,0 +1,805 @@
+                                showIcon={false}
+                                stroke={true}
+                                style={{ background: "#EFF8FF", height: "fit-content", maxWidth: "100%" }}
+                                className={"version-tag"}
+                                labelStyle={{ whiteSpace: "normal", wordBreak: "break-word" }}
+                            />
+                            {/* Actions row BELOW, right aligned */}
+                            <div style={{ display: "flex", justifyContent: "flex-end", alignItems: "center", gap: "0.75rem" }}>
+                                {/* Edit */}
+                                <div
+                                    role="button"
+                                    title={editLabel}
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<button>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)